### ISSUE TITLE
request

### ISSUE THREAD

**Issue by erdoukki** [fln/addrwatch#15]
Like it was possible with arpwatch, is it possible to get email when new network devices are discovered ?

Any sample of this type of configuration available ?

**Comment by fln** [comment]
Addrwatch works a bit different than arpwatch. Addrwatch records device activity on the network as it happens. It allows to detect not only existence of the device but also show its activity over time. On the other hand arpwatch tries to keep a list of known devices and supports alerting only when new device is discovered.

Adding a feature to send email any time a device activity is detected will lead to a lot of duplicate email messages. And that is probably not what you would expect.

It is possible su suppress duplicates for some time but active device list is never persisted between service restarts.

If receiving email alerts on device activity is what you expect then email alerting could be added as a new addrwatch output module.

**Comment by erdoukki** [comment]
> If receiving email alerts on device activity is what you expect then email alerting could be added as a new addrwatch output module.

Yes, it is just what I need...

**Comment by a-wilkinson** [comment]
Yes I would absolutely love a new email addrwatch(1) output module also. Pretty please add ASAP :-)
